src/main/java/net/kyori/adventure/text/minimessage/transformation/inbuild/DecorationTransformation.java

```diff
@@ -73,28 +77,39 @@
    * @return a new transformation
    * @since 4.2.0
    */
-  public static DecorationTransformation create(final String name, final List<TagPart> args) {
+  public static DecorationTransformation create(String name, final List<TagPart> args) {
+    boolean flag = args.size() != 1 || !args.get(0).isFalse();
```

Comment:
Do we want to throw an error for bad arguments? For example, `<b:hello>` would be okay here.